### PR TITLE
migrate patientrole to main fhir api route

### PR DESCRIPTION
**Pull request by bradymiller**
just experimenting with way to securely migrate patient role to use the main fhir api

### CODE REVIEW

File: src/RestControllers/AuthorizationController.php [hunk 1]
```diff
@@ -661,7 +661,7 @@ public function userLogin(): void
     {
         $response = $this->createServerResponse();
 
-        $patientRoleSupport = (!empty($GLOBALS['rest_portal_api']) || !empty($GLOBALS['rest_portal_fhir_api']));
+        $patientRoleSupport = (!empty($GLOBALS['rest_portal_api']));
```
Comment (bradymiller) [inline src/RestControllers/AuthorizationController.php:664]: need fhir global here also

**Comment by bradymiller** [comment]
getting closer on this. note still experimental (i have not even tested it yet :) )

**Comment by adunsulag** [comment]
@bradymiller This is looking good, I'd like to take this and build on it for the SMART standalone.  I'm going to try and pull it into my branch if you're ok with that.

**Comment by bradymiller** [comment]
go for it (i still have testing to do to ensure, but can do that while your
working with it). also note i changed the logger class from a singleton to
a standard class since things were breaking (the global settings were not
be followed) if the class was used before the globals.php context was set.

**Comment by bradymiller** [comment]
pertinent code in https://github.com/openemr/openemr/pull/4177

**Pull request closed without merging**